go/types/indexed_sequences.go

```diff
@@ -59,22 +59,6 @@ func (ims indexedMetaSequence) getCompareFn(other sequence) compareFn {
 	}
 }
 
-func newCursorAtIndex(seq indexedSequence, idx uint64) *sequenceCursor {
```

Comment:
It now works with all sequences, not just indexed sequences, so I moved it to sequence_cursor.go.